### DISCUSSION TITLE
warden svc up  cannot bind to port 53 or 443

### DISCUSSION THREAD

**Discussion by ERJulian** [wardenenv/warden#790] [Help & Support]
### Version of Warden

in-dev

### Operating System and Installation Method

EndeavourOS via alternative/manual installation

### Describe the Bug

warden svc up
[+] Running 2/0
 ✔ Container mailhog  Running                                                                                                                                                             0.0s
 ⠋ Container dnsmasq  Starting                                                                                                                                                            0.0s
 ⠋ Container traefik  Starting                                                                                                                                                            0.0s
 ✔ Container tunnel   Started                                                                                                                                                             0.0s
Error response from daemon: Ports are not available: exposing port UDP 127.0.0.1:53 -> 0.0.0.0:0: listen udp 127.0.0.1:53: bind: permission denied

this is the output, I've tried with --force-recreate, still no luck

### To Reproduce

1. Download warden manually
2. Run docker
3. execute `warden svc up`
4. Error

### Expected Behavior

It should start warden services without errors

**Comment by bap14** [comment]
@ERJulian Lower numbered port numbers (port numbers less than 1024) require admin/super user privileges to bind. Given the error "permission denied" it seems that the underlying Docker system isn't provided the privileged access it needs to bind to port 53. It may also fail to bind to ports 80 and 443 for the same reason.

I'd take a look at what you need to allow your Docker installation to access privileged ports.

**Comment by ERJulian** [comment]
@bap14 had to do this `sysctl -w net.ipv4.ip_unprivileged_port_start=20` and it solved the issue though I'm sure that's not the best thing to do, but kindof in a hurry so
